src/framework/Cron.ts

```diff
@@ -62,6 +65,24 @@ export class Cron extends Base {
       await this.handler({ date, client: bot.client, logger });
     } catch (error) {
       logger.error(error, 'cron handler error');
+      try {
+        await findTextChannelByName(bot.client.channels, 'logs').send(
+          new MessageEmbed()
+            .setTitle('Cron run failed')
+            .addField('Name', this.name, true)
+            .addField('Run id', id, true)
+            .setDescription(
+              dedent`
+              \`\`\`
+              ${error.stack}
+              \`\`\`
+            `,
+            )
```

Comment:
Il y a t-il réellement besoin de dedent ? 
```suggestion
            .setDescription(`\`\`\`\n${error.stack}\`\`\``)
```

Reply:
Plus maintenant, mais je pense que le module peut être utile dans le futur et je trouve plus clair d'utiliser de vrais retours à la ligne plutôt que `\n`

Reply:
Je pense qu'on peut le réinstaller quand on en aura besoin. Avec le passage à discord.js 13 dans #34, les `Formatters` ont été introduits pour remplacer ça par `Formatters.codeBlock(error.stack)`, donc dedent n'a plus d'utilité pour le moment, imo

Reply:
c'est enlevé